WBooks/DetailBook/DetailBookViewModel.swift

```diff
@@ -11,83 +11,21 @@ import Alamofire
 class DetailBookViewModel {
     
     let book: Book
-    var comments: [Comment]
     
     init(book: Book) {
         self.book = book
-        self.comments = []
     }
     
-    func getComments(action: @escaping ([Comment]) -> () ) {
-        
-        let onSuccess = { (comments: [Comment]) in
-            self.comments = comments
-            action(comments)
-        }
-                
-        let onError = { error in
-            debugPrint(error)
-        }
-        
-        let repository = CommentsRepository()
-        repository.fetchComments(id: book.id, onSuccess: onSuccess, onError: onError)
+    func createDetailViewModel() -> DetailViewModel {
+        return DetailViewModel(book: book)
     }
     
-    func getUserComment(id: Int, action: @escaping (User) -> () ) {
-        
-        let onSuccess = { (user: User) in
-            action(user)
-        }
-                
-        let onError = { error in
-            debugPrint(error)
-        }
-        
-        let repository = UserRepository()
-        repository.getUser(id: id, onSuccess: onSuccess, onError: onError)
+    func createCommentsViewModel() -> CommentsViewModel {
+        return CommentsViewModel(bookId: book.id)
```

Comment:
If the function only has one line, we can avoid the `return`

Reply:
Well, thank you for the observation, I will keep this adjustment in mind for the next pr 😄